packages/components/organisms/f-checkout/src/components/_tests/ErrorDialog.test.js

```diff
@@ -76,24 +81,6 @@ describe('ErrorDialog', () => {
             });
         });
 
-        describe('restaurantMenuPageUrl ::', () => {
```

Comment:
Moved the test to Checkout.tests.js as now the computed prop lives there and not in the ErrorDialog.vue